addons/website_event/controllers/main.py

```diff
@@ -74,7 +74,16 @@ def events(self, page=1, **searches):
         events = events[(page - 1) * step:page * step]
 
         # count by domains without self search
-        domain_search = [('name', 'ilike', fuzzy_search_term or searches['search'])] if searches['search'] else []
+        domain_search = []
+        if searches['search']:
+            address_event_ids = Event.sudo().search([
+                ('address_search', 'ilike', fuzzy_search_term or searches['search']),
+            ]).ids
+            domain_search = [
+                '|',
+                ('name', 'ilike', fuzzy_search_term or searches['search']),
+                ('id', 'in', address_event_ids),
+            ]
```

Comment:
Yes, if you revert, it works, but only when you are logged in; it does not work for public users.

Reply:
ah ok :) can we add a test about it ? (both public and logged user)

Reply:
`> Yes, if you revert, it works, but only when you are logged in; it does not work for public users.`
Please ignore this.

Yes is it work if revert this.

Reply:
`> Yes, if you revert, it works, but only when you are logged in; it does not work for public users.`
Please ignore this.

Yes is it work if revert this.

Reply:
Yes, if we revert this, the search will work properly. because we include address_search in the search_get_detail method when it calls website._search_with_fuzzy.